resources/function_help/json/straight_distance_2d

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "straight_distance_2d",
+  "type": "function",
+  "groups": ["GeometryGroup"],
+  "description": "Returns the straight distance of a geometry. The geometry must be a curve (circularstring, linestring).",
```

Comment:
```suggestion
  "description": "Returns the direct/euclidean distance between the first and last vertex of a geometry. The geometry must be a curve (circularstring, linestring).",
```
I think we better explain what this straightdistance represents. Picked at https://qgis.org/api/3.20/classQgsCurve.html#ac40baa1bf589ed744e7aa56ac3580f63